elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideResourceConfig.java

```diff
@@ -51,30 +53,52 @@
     private static HealthCheckRegistry healthCheckRegistry = null;
 
     /**
-     * Constructor
+     * Constructor.
      *
-     * @param injector Injection instance for application
+     * @param injector Injection instance for application.
+     * @param servletContext servlet context instance.
      */
     @Inject
     public ElideResourceConfig(ServiceLocator injector, @Context ServletContext servletContext) {
         this.injector = injector;
 
         settings = (ElideStandaloneSettings) servletContext.getAttribute(ELIDE_STANDALONE_SETTINGS_ATTR);
 
+        if (settings.enableDynamicModelConfig()) {
+            try {
+                Util.initDynamicConfig(settings.getDynamicConfigPath());
+            } catch (Exception e) {
+                log.error("error in resource config" + e.getMessage());
+                throw new IllegalStateException(e);
+            }
+        }
+
         // Bind things that should be injectable to the Settings class
         register(new AbstractBinder() {
             @Override
             protected void configure() {
-                bind(Util.combineModelEntities(settings.getModelPackageName(), settings.enableAsync())).to(Set.class)
-                        .named("elideAllModels");
+                try {
+                    bind(Util.combineModelEntities(Util.dynamicEntityCompiler,
+                            settings.getModelPackageName(), settings.enableAsync(),
+                            settings.enableDynamicModelConfig())).to(Set.class).named("elideAllModels");
+                } catch (ClassNotFoundException e) {
+                    log.error("error in resource config" + e.getMessage());
+                    throw new IllegalStateException(e);
+                }
             }
         });
 
         // Bind to injector
         register(new AbstractBinder() {
             @Override
             protected void configure() {
-                ElideSettings elideSettings = settings.getElideSettings(injector);
+                ElideSettings elideSettings = null;
+                try {
+                    elideSettings = settings.getElideSettings(injector);
+                } catch (Exception e) {
```

Comment:
What exception are we catching here an why?

Reply:
The DynamicEntityompiler compile method throws an Exception so this was catching and throwing that. Have updated DynamicCompiler to catch it and throw a run time exception now instead of compile time.